src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Get condition recommendations
+metaDescription: For APM entities, you can see and create recommended alert conditions.
+watermark: Limited release
+---
+
+Use the New Relic Navigator's mini-overview to get alert condition recommendations for your monitored entities. With these recommendations, you can create alerts conditions to be notified when your entities aren't performing as they should.
+
+<Callout variant="caution">
+Condition recommendations are only available for APM-monitored entities.
+</Callout>
+
+## Why it matters
+
+Condition recommendations are all about making it easier for you to find entities and their conditions and thresholds you'd want to be notified about.
```

Comment:
Does this feature help users find entities and their conditions, or does it help the user by suggesting conditions and thresholds for entities?

Reply:
It doesn't help users find entities. Once you've found an (APM-monitored) entity and look at its mini-overview, you can see its recommended conditions.